website/views/shared/tasks/lists.jade

```diff
@@ -24,9 +24,14 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', required)
+            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', required)
+            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
             button(type='submit', ng-disabled='new{{list.type}}form.$invalid')
               span.glyphicon.glyphicon-plus
+            span.help-block
```

Comment:
I thought `.help-block` looked familiar. How about this then:
```
small.help-block.btn-link.pull-right(ng-click='toggleBulk(list)')
  span(ng-if='!list.bulk') Add Multiple
  span(ng-if='list.bulk') Add Single
```

Including the `.help-block` class afterall is good, because it includes a `margin-top: 5px`. Making the element `small` and adding the `btn-link` and `pull-right` classes eliminate the need for the extra styles in `tasks.styl`.

Does that seem sound to you?